src/Profiles.ts

```diff
@@ -446,141 +440,29 @@ export class Profiles {
         }
     }
 
-    public async editSession(profileLoaded: IProfileLoaded, profileName: string): Promise<any| undefined> {
-        const editSession = profileLoaded.profile;
-        const editURL = editSession.host+ ":" + editSession.port;
-        const editUser = editSession.user;
-        const editPass = editSession.password;
-        const editrej = editSession.rejectUnauthorized;
-        let updUser: string;
-        let updPass: string;
-        let updRU: boolean;
-        let updPort: number;
-        let updUrl: any;
-
-        const schema: {} = await this.getSchema(profileLoaded.type);
-        const schemaArray = Object.keys(schema);
-
-        const updSchemaValues: any = {};
+    public async editSession(profileLoaded: IProfileLoaded, profileName: string): Promise<IProfile | void> {
+        const schema = await this.getSchema(await this.getProfileType());
```

Comment:
Can you put this back? const schema: {} = await this.getSchema(profileLoaded.type);
This will fix the configurations of null error in edit session. The profileType is already added in profileLoaded so there is no need to prompt it.

Reply:
Ok I changed what I had done to fix it...was searching for something like this but couldn't find it ^^'